ui/packages/consul-peerings/app/components/consul/peer/list/test-support.js

```diff
@@ -10,9 +10,10 @@ export const selectors = {
     },
   }
 };
-export default (collection, isPresent, attribute) => () => {
+export default (collection, isPresent, attribute, actions) => () => {
   return collection(`${selectors.$} ${selectors.collection.$}`, {
     peer: isPresent(selectors.collection.peer.$),
     name: attribute('data-test-peer', selectors.collection.peer.name.$),
+    ...actions(['regenerate', 'delete']),
```

Comment:
Mind me asking what this does? Is this related to the `regenerate` part of the `And I click regenerate on the peers` step?

Reply:
I love you asking what this does! Yes exactly, its related to that step `And I click regenerate on the peers`

Umm briefly

the `...actions()` page object helper lets you configure actions easily, actions being the little 3 dot 'meatball' menus we have at the end of every row. Peer listing rows can have `regenerate` and `delete` actions, so here we configure the page object to reflect that. Once that is done you can refer to these actions in the tests:

`And I click regenerate on the peers`

If I'd done `...actions(['edit', 'delete'])` (and there was a corresponding data-test-edit on the dom element) then I would have had to write:

`And I click edit on the peers`

Does that help?

Reply:
Yep I see it now. I see the `morePopoverMenu` page object. Was confused at first how it was selecting it. 

Thanks!